### PR TITLE
Heroku One Click Deploy Patched

### PR DESCRIPTION
**Pull request by phoenix1796**
The previous Heroku deploy using the button did not work properly because it didn't list out all the steps like in doc/heroku.md 
I have added the following :
- Addons:
    - Postgres:hobby-dev
    - Redis:hobby-dev
- Buildpacks (In same order as mentioned in the docs)

(Above changes are in app.json)
This should fix the one click heroku button deploy

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -19,6 +19,18 @@
...
+  "addons":[
+      "heroku-postgresql:hobby-dev",
+      "heroku-redis:hobby-dev"
+  ],
+  "buildpacks":[
+      {
+          "url":"heroku/nodejs"
+      },
+      {
+          "url":"heroku/python"
```
Comment (niranjan94) [inline app.json]: Could you swap the order of the build packs ? Python first and nodejs next. I'll update the docs.

**Comment by phoenix1796** [comment]
@niranjan94 I have added the buildpacks in the same order as that written in docs/heroku.md
this is the correct order because the last buildpack decides the process types in the procfile

**Comment by niranjan94** [comment]
@phoenix1796 yes. You're right about the last build pack determining the process type. But that's why we have a 'Procfile' in the project root to tell heroku what the process is. 

Right now both our heroku deployments have python first and nodejs next. So, could you swap the order?

**Comment by phoenix1796** [comment]
@niranjan94 Done , Buildpacks swapped : )

**Pull request merged**